addons/website/static/src/components/fields/fields.xml

```diff
@@ -36,4 +36,14 @@
     <span><i t-attf-class="fa fa-fw {{prefix.class}} {{prefix.visible ? '': 'invisible'}}" t-att-title="prefix.help"/><t t-esc="_formatValue(value)"/></span>
 </t>
 
+<t t-name="website.FieldImageRadio" owl="1">
+    <div class="d-flex my-4">
```

Comment:
Does the design of other fields also use random bootstrap classes for spacing? Is not there a common spacing class for all fields? :grimacing:

Reply:
As far as I know, there is no specific spacing class, I did a quick code check and couldn't find any (e.g. [Boolean field here](https://github.com/odoo/odoo/blob/99f7d9e7f1301f231cd61a36f4171234b44d450b/addons/web/static/src/views/fields/boolean/boolean_field.xml#L5) uses a `CheckBox` component with BS spacing...)

Reply:
Randomly passing by, this should IMHO be part of the `FieldImageRadio` design (`.o_image_radio_option`).
As those receive a box shadow, everyone will want to make that shadow not going "against" the edge of the next element, so everyone will add the same margin everywhere as you did here.

Adding a margin bottom & right on that element default style should be enough to remove the need of adding those custom margin afterward, and it will also fit the style of other fields (we should probably never go against the default style and add margin like that because we think it is more beautiful, as [the last time here](https://github.com/odoo/odoo/pull/95949#discussion_r931191539))

Best to keep the design the framework gives us for the sake or consistency across the software.
If we think there is not enough space between fields, that's either:
- True and need to be suggested/processed at the framework level
- False and just a matter of personal preference

Except in this case where this is a custom field we added ourselves, so IMHO it just need a final generic visual fine tuning.